node/node.go

```diff
@@ -168,6 +175,29 @@ func Setup(ctx context.Context, init NodeInitializer) (*Node, error) {
 	}
 
 	h, err := libp2p.New(opts...)
+
+	//	peering service
+	hPeer := peering.NewEstuaryPeeringService(h)
+
+	peeringPeerList := cfg.PeeringPeers
+	peeringPeerList = append(peeringPeerList, bootStrapPeeringPeers...)
+
+	//	add the peers
+	for _, addrInfo := range peeringPeerList {
+		addrs, err := toMultiAddresses(addrInfo.Addrs)
+		if err != nil {
+			return nil, fmt.Errorf("failed to parse peering peers multi addr: %w", err)
+		}
+		addrInfoId, err := peer.Decode(addrInfo.ID)
+		if err != nil {
+			return nil, fmt.Errorf("failed to parse peering peers multi addr ID: %w", err)
+		}
+		hPeer.AddPeer(peer.AddrInfo{ID: addrInfoId, Addrs: addrs})
+	}
+
+	//	We only want to start if there are peering_peers configured
```

Comment:
Maybe also add this:
```
err := hPeer.Start()
if err != nil {
    log.Warnf(err)
}

```

Reply:
@en0ma I initially did it this way but I think we should still start the peering regardless if there are pre-defined peers or not.

Reply:
@alvin-reyes I only made the suggestion because of your comment `//	We only want to start if there are peering_peers configured` - ignore my suggestion and remove the comment